avm/ptn/app/iaas-vm-cosmosdb-tier4/DIAGRAM.md

```diff
@@ -0,0 +1,127 @@
+# Architecture Diagram
```

Comment:
Not a **must**, but I'd recommend adding the content of this file to the README.md file. This files is generated from scratch each time - but - if you add your content under a header `## Notes`, you should be good to go.